docs/root/index.rst

```diff
@@ -20,3 +23,11 @@ Envoy documentation
   api/api
   faq/overview
   version_history/version_history
+
+Developer documentation
+-----------------------
+
+.. toctree::
+  :maxdepth: 2
+
+  dev/index
```

Comment:
I could go either way on this ultimately, but my suggestion from before was to make this hidden, and then just have a direct link to the dev docs root from our README. I think this would avoid polluting the end-user docs with dev docs while still allowing cross linking.

Reply:
at least with the redirects plugin in we have now this is not possible,  and wouldnt be without quite a lot of hackery

i searched again today to see if there was  anything like a "split-tree" rendering in sphinx and turned up nothing

i think to go the way you suggest is fighting the system too much, altho  i  guess its  theoretically possible

Reply:
OK it's not a big deal. Maybe just set max depth to 1 or something so not much shows up in the main index?

Reply:
yep, sgtm